react/core/main.tsx

```diff
@@ -245,6 +245,7 @@ function start() {
 
       if (type === 'img') {
         props.src = optimizeSrcForVtexImg(vtexImgHost, props.src)
+        props.loading = 'lazy'
```

Comment:
wasn't an attempt of doing this causing issues in the past?

Reply:
Nope, I used a third party crazy lib. This is a native Google Chrome feature!